examples/siro_sandbox/sandbox_app.py

```diff
@@ -937,15 +962,40 @@ def parse_debug_third_person(args, framebuffer_size):
             )
             task_config.actions.arm_action.arm_controller = "ArmEEAction"
 
-    assert (
-        args.gui_controlled_agent_index is None
-        or args.gui_controlled_agent_index >= 0
-        and args.gui_controlled_agent_index
-        < len(config.habitat.simulator.agents)
-    ), (
-        f"--gui-controlled-agent-index argument value ({args.gui_controlled_agent_index}) "
-        f"must be >= 0 and < number of agents ({len(config.habitat.simulator.agents)})"
-    )
+        if args.gui_controlled_agent_index is not None:
+            assert (
```

Comment:
Best practices for Python IMO is we should only use asserts for cases we *never* expect to happen. For routine errors coming from bad user input, an exception is better. See discussion here: https://stackoverflow.com/questions/944592/best-practice-for-using-assert

An example of how this matters: one can disable asserts as a (minor) runtime perf optimization; your program should still work correctly in that case.